spec/features/posts_appear_newest_first_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Timeline", type: :feature do
+  let!(:create_posts) do
```

Comment:
last change. This feels like it should either be extracted to a helper method or split into two `let1`s because that is what the #create method is returning:

```ruby
let!(:post1)  { create(:post, message: "First post") }
let!(:post2)  { create(:post, message: "Second post") }
```

However, ideally i'd like to see something that does this:

```ruby
let!(:post1)  { create(:post }
let!(:post2)  { create(:post }
```
Which means we would have to have someway of generating a string perhaps
`FactoryGirl's #sequence`